contrib/slime-cl-indent.el

```diff
@@ -1729,6 +1729,7 @@ Cause subsequent clauses to be indented.")
              (defconstant (as defvar))
              (defcustom   (4 2 2 2))
              (defparameter     (as defvar))
+             (define-compiler-macro (as defun))
```

Comment:
I think these are supposed to be sorted alphabetically. If you find others that are out of order and want to sort them please make that a different commit.

Reply:
Done